catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/RegistryRestEndpoint.java

```diff
@@ -55,7 +54,7 @@
 
     private RegistryPublicationService registryPublicationService;
 
-    private RegistryReportMapBuilder reportMapBuilder;
+    private RegistryReportBuilder registryReportBuilder;
 
     public RegistryRestEndpoint() {
         templateLoader = new ClassPathTemplateLoader();
```

Comment:
It was there originally, I can move it over to the RegistryReportBuilder now since it is not really being utilized anywhere else in RegistryRestEndpoint